cdap-app-fabric/pom.xml

```diff
@@ -149,6 +149,10 @@
       <artifactId>MockFtpServer</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-service</artifactId>
```

Comment:
Yes. I did the changes but then we had to more TokenSecureStoreUpdater class from cdap-master to cdap-app-fabric as AbstractDistributedProgramRunner needs TokenSecureStoreUpdater now and master already depends on app-fabric. The TokenSecureStoreUpdater uses more constants from Hive so we just thought it will better to add hive-service dependency. 
https://github.com/caskdata/cdap/pull/4738/files#diff-75161f7fea7e589954bceab3ebba8b38R147
Thoughts ?